addon/materials_manager.py

```diff
@@ -0,0 +1,35 @@
+# ******************************************************************************/
```

Comment:
I think this code should go together with the other helpers. No separate file needed. Otherwise we at least need the GPL license header here instead of the provided header.

Reply:
I'm sorry I didn't notice that. i have changed it.